apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3480,9 +3657,16 @@ handle_call_(St, settle, From, D) ->
         channel_closing ->
             {ok, SettleTx, Updates, BlockHash} = settle_tx_for_signing(D),
             gen_statem:reply(From, ok),
-            D1 = set_ongoing(request_signing(settle_tx, SettleTx, Updates,
-                                             BlockHash, D)),
-            next_state(awaiting_signature, D1);
+            case request_signing(settle_tx, SettleTx, Updates, BlockHash, D, defer) of
```

Comment:
This whole case statement is repeated quite often, maybe it should be refactored into its own function.

Reply:
Maybe ...

Reply:
I should watch how I phrase feedback ;-)